.github/TOPIC_TEMPLATE

```diff
@@ -0,0 +1,222 @@
+---
+title: Basic template
+---
+
+Introductory text that gives an overview of the topic you will be writing about.
+
+The purpose of this page is to provide you with a pre-formatted template and useful [markdown](https://glossary.magento.com/markdown) references to help you get started writing docs.
+
+You can start off by editing the local version of this file using markdown language (and [HTML](https://glossary.magento.com/html) where needed). Then, create a Pull Request to have your contribution reviewed by the Docs team.
+
+## Metadata parameters
+
+Add the following metadata parameters at the top of your page. We use YAML for the metadata in front matter. We use this data when generating the DevDocs for the following:
+
+Parameter | Description
+--- | ---
+`title:` | The title of the page.
+`group:` | Optional. The table of contents this file belongs to if it does not follow the default for its directory. The group maps to files located in `_data/toc/`. Only add the name of the file without the extension. For example, if the group of the file is `group: getting-started`, it points to the table of contents defined in `_data/toc/getting-started.yml`.
+`redirect_from` | Optional. Add a list of other pages in the User Guide that should redirect to this page.
+`ee_only:` | Optional. If set to `true`, graphics/cues display on the page indicating it applies to the Magento Commerce product and is not included with Open Source installations.
+`b2b_only:` | Optional. If set to `true`, graphics/cues display on the page indicating it applies to the Magento Commerce product with the B2B extension installed and configured.
+
+If you need help with metadata, we can assist!
+
+## Basic markdown syntax {#basic}
+
+Below are some basic examples of what you can do with markdown.
+
+### Text effects {#text}
+
+| Example           | Output       |
+| ----------------- | ------------ |
+|`_emphasis_`       | _emphasis_   |
+|`**bold**`         | **bold**     |
+|`` `inline code` ``| `inline code`|
+
+Use code fences to create code blocks. For extensive examples of adding code samples, see [Code blocks](#code-blocks).
+
+```text
+//This is a code block!
+print "Hello World!";
+```
+
+For more examples of basic markdown please follow this [link](https://daringfireball.net/projects/markdown/syntax){:target="_blank"}.
+
+### Lists {#lists}
+
+Lists are useful for organizing and displaying related items. Below are examples of a bulleted list and an ordered list.
+
+#### Bulleted list
+
+```markdown
+- List Item 1
+- List Item 2
+- List Item 3
+```
+
+Output:
+
+-  List Item 1
+-  List Item 2
+-  List Item 3
+
+#### Ordered list
+
+```markdown
+1. First Step
+1. Second Step
+1. Third Step
+```
+
+Output:
+
+1. First Step
+1. Second Step
+1. Third Step
+
+### Images {#images}
+
+Please add any images you may need to the [`src/images`](https://github.com/magento/merchdocs/tree/master/src/images) directory.
```

Comment:
```suggestion
Add any images you may need to the [`src/images`](https://github.com/magento/merchdocs/tree/master/src/images) directory.
```

also, don't some projects use the Assets folder?

Reply:
For now that is the primary place. We can adjust location for now if there is an assets folder.